oai-pmh.rdf

```diff
@@ -9,22 +9,21 @@
     <owl:Ontology rdf:about="http://fedora.info/definitions/v4/oai-pmh#">
         <rdfs:label xml:lang="en">Fedora Commons OAI-PMH Ontology</rdfs:label>
         <rdfs:comment xml:lang="en">Describes relationships between Fedora Commons Repository objects used to provide an OAI-PMH service.</rdfs:comment>
-        <owl:versionInfo>v4/2015/03/26</owl:versionInfo>
-        <owl:priorVersion rdf:resource="http://fedora.info/definitions/v4/2014/11/27/oai-pmh"/>
+        <owl:versionInfo>v4/2017/02/17</owl:versionInfo>
+        <owl:priorVersion rdf:resource="http://fedora.info/definitions/v4/2015/03/26/oai-pmh"/>
     </owl:Ontology>
 
     <owl:ObjectProperty rdf:about="http://fedora.info/definitions/v4/oai-pmh#isPartOfOAISet">
         <rdfs:comment xml:lang="en">Links a Fedora object to an OAI set.</rdfs:comment>
-        <rdfs:domain rdf:resource="http://fcrepo.org/repository#object"/>
```

Comment:
Sure! There's no reason a predicate can't link between to entities of the same kind.